subsys/net/lib/aws_fota/include/aws_fota_json.h

```diff
@@ -27,12 +27,47 @@ extern "C" {
  */
 #define STATUS_MAX_LEN (12)
 
-int aws_fota_parse_notify_next_document(char *job_document,
-		u32_t payload_len, char *job_id_buf, char *hostname_buf,
-		char *file_path_buf);
+/**
+ * @brief Parse a given AWS IoT DescribeJobExecution response JSON object.
+ *	  More information on this object can be found at https://docs.aws.amazon.com/iot/latest/developerguide/jobs-api.html#mqtt-describejobexecution
+ *
+ * @param[in] job_document  JSON formated string of an AWS IoT
+ *			    DescribeJobExecution response.
+ * @param[in] payload_len  The length of the provided string.
+ * @param[out] job_id_buf  Output buffer for the "jobId" field from the
+ *			   JobExecution data type.
+ * @param[out] hostname_buf  Output buffer for the "host" field from the Job
+ *			     Document
+ * @param[out] file_path_buf  Output buffer for the "file" field from the Job
+ *			      Document
+ * @param[out] version_number  Version number from the Job Execution data type.
+ *
+ * @return 0 if the Job Execution object is empty, 1 if Job Execution object was
+ *	     correctly decoded, otherwise a negative error code is returned
+ *	     identicating reason of failure.
+ **/
+int aws_fota_parse_DescribeJobExecution_rsp(char *job_document,
+					    u32_t payload_len,
+					    char *job_id_buf,
+					    char *hostname_buf,
+					    char *file_path_buf,
+					    int *version_number);
 
-int aws_fota_parse_update_job_exec_state_rsp(char *update_rsp_document,
-		size_t payload_len, char *status);
+/**
+ * @brief Parse a Job Execution accepted response object. Returned by a call to
+ *	  https://docs.aws.amazon.com/iot/latest/developerguide/jobs-api.html#mqtt-updatejobexecution
+ *
+ * @param[in] job_document  JSON formated string of an AWS IoT Job execution
+ *			    update response.
+ * @param[in] payload_len  The length of the provided string.
+ * @param[out] status  String with the AWS Job status after an update.
+
+ * @return 0 for an successful parsing or a negative error code identicating
```

Comment:
I think this 0 is wrong as well (bits have been set).

Reply:
Yup. fixed.